mon-pix/translations/en.json

```
@@ -1419,6 +1420,14 @@
       },
       "not-finished": "You can’t submit your results yet, we still have a few questions to ask.",
       "organization-message": "Message of your organization",
+      "reset": {
+        "button": "Reset and restart from the beginning",
+        "modal": {
+          "text": "You are about to reset and restart your customized test <strong>{targetProfileName}</strong>, in order to improve your result. At the end of the customized test, you can resend your results.",
```

Bon - je suis pas trop fan de ce phrase (le premier), mais je suis pas trop sur comment d'ameliorer
```suggestion
          "text": "You are about to reset and restart your customized test <strong>{targetProfileName}</strong>, in order to try to improve your result. At the end of the customized test, you can resend your results.",
```
ou
```suggestion
          "text": "You are about to reset and restart your customized test <strong>{targetProfileName}</strong>, to try and improve your result. At the end of the customized test, you can resend your results.",
```
ou
```suggestion
          "text": "To try and improve your result, you are about to reset and restart your customized test <strong>{targetProfileName}</strong>. At the end of the customized test, you can resend your results.",
```